nlp_profiler/high_level_features/grammar_quality_check.py

```diff
@@ -1,6 +1,5 @@
-import language_tool_python
-
-language_tool = language_tool_python.LanguageTool('en-GB')
+from gingerit.gingerit import GingerIt
```

Comment:
minor but imp: please apply some formatting - run `black` on your code changes

Reply:
See https://pypi.org/project/black/ to install

Reply:
hello 

> 

hello i have added comments and cleaned the code a bit more

Reply:
https://www.freecodecamp.org/news/auto-format-your-python-code-with-black/